packages/mobile/lib/pincodeManager.ts

```diff
@@ -0,0 +1,51 @@
+import 'capacitor-secure-storage-plugin';
+import { Plugins } from "@capacitor/core"
+
+const { SecureStoragePlugin } = Plugins
+
+function handle_keychain() {
+    
+    function set(key: string, pincode: string): Promise<boolean> {
+        return SecureStoragePlugin.set({ 
+            key, 
+            value: pincode 
+        }).catch(error => {
+            console.trace({ error })
```

Comment:
We should explicitly throw the error here, otherwise the caller promise chain can't won't be able to detect if an error occurs here.

Reply:
That's good point, sorry I've just let this here for testing while writing, I have had to delete it. To build on that I've saw that the code of `pincodeManager.js` and `keychain.js` on desktop folder just returns the promise, since erros are handled by module itself. Do you think I can make the same as in desktop and just return the promise?